app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -255,18 +288,19 @@ class WeightedQueryStrategy
       def initialize(user: nil, number_of_articles: 50, page: 1, tag: nil, **config)
         @user = user
         @number_of_articles = number_of_articles.to_i
-        @page = page.to_i
+        @page = (page || 1).to_i
         # TODO: The tag parameter is vestigial, there's no logic around this value.
         @tag = tag
         @default_user_experience_level = config.fetch(:default_user_experience_level) { DEFAULT_USER_EXPERIENCE_LEVEL }
         @negative_reaction_threshold = config.fetch(:negative_reaction_threshold, DEFAULT_NEGATIVE_REACTION_THRESHOLD)
         @positive_reaction_threshold = config.fetch(:positive_reaction_threshold, DEFAULT_POSITIVE_REACTION_THRESHOLD)
-        @oldest_published_at = determine_oldest_published_at(
-          user: @user,
-          most_number_of_days_to_consider: config.fetch(:most_number_of_days_to_consider, 31),
-        )
         @scoring_configs = config.fetch(:scoring_configs) { default_scoring_configs }
         configure!(scoring_configs: @scoring_configs)
+
+        @oldest_published_at = self.class.oldest_published_at_to_consider_for(
+          user: @user,
+          published_since: @published_since,
```

Comment:
it was slightly surprising that `@published_since` is referenced but not visible - I see it's setup in `configure!` (earlier, but further down in the file).